### ISSUE TITLE
hightlight read

### ISSUE THREAD

**Issue by privat** [privat/rars#7]
Memory and registers are highlighted on write events, but read events are missing.

* `add a0, a1, a2` highlight `a0` in the Register pane but `a1` and `a2` are not highlighted.
* `lw a0, 0(sp)`  highlight the `a0` but not the address in the segment window, nor `sp`.

We want read event to also highlight memory and registers.
Two different colors should be used (pink for write and blue for read or whatever you like) this could be configurable.

Note that on a single instruction, multiple register or memory cells cans be read or write, all colors should be present.
If a same register is read and write, the write event can win I suppose :)

Related: #6

**Comment by ENDERZOMBI102** [comment]
> Was this resolved by [#34](https://github.com/rarsm/rars/pull/34) ?

yes, register highlighting is working with reads, this simple program produces blue (read) and red (write) highlights

```asm
.text
	and s2, sp, sp
```
